Binary files orig/PKI/src/.Makevars.win.swp and patched/PKI/src/.Makevars.win.swp differ
diff -Nru orig/PKI/src/Makevars.win patched/PKI/src/Makevars.win
--- orig/PKI/src/Makevars.win	2022-11-27 22:31:11.000000000 +0100
+++ patched/PKI/src/Makevars.win	2023-10-10 09:31:58.206511500 +0200
@@ -1 +1,6 @@
-PKG_LIBS = -lssl -lcrypto -lws2_32 -lgdi32 -lcrypt32 -lz
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_LIBS = -lssl -lcrypto -lws2_32 -lgdi32 -lcrypt32 -lz
+else
+  PKG_LIBS = $(shell pkg-config --libs openssl)
+endif
+
